internal/provisioner/registry/README.md

```diff
@@ -1,3 +1,278 @@
-# registry
+# Registry Provisioner
 
-This provisioner was removed in commit 76fefcfcd1408d04b4eb7eb368d3eaab49565237.
+## Summary
+
+The `registry` provisioner is a core Rukpak provisioner that knows how to interact with bundles of a particular format.
+These `registry+v1` bundles, or `registry` bundles, are container images containing a set of static Kubernetes YAML
+manifests organized in the legacy Operator Lifecycle Manger (OLM) format. For more information on the `registry+v1` format, see
+the [OLM packaging doc](https://docs.openshift.com/container-platform/4.10/operators/understanding/olm-packaging-format.html).
+
+The `registry` provisioner is able to convert a given `registry+v1` bundle onto a cluster in the `plain+v0` format. Instantiation of the
```

Comment:
Somewhere in here we might want to list the limitations of the conversion. We don't support _all_ registry+v1 bundles.

Reply:
I left a new note that outlines this particular issue.